lib/tasks/duplicates_metadata.rake

```diff
@@ -39,3 +24,52 @@ end
 def logger_output(msg = "", task_name = "decidim:duplicates:metadata")
   "[#{task_name}] :: #{msg}"
 end
+
+def update_socio_authorized_users(socio_authorizations)
```

Comment:
I think we should think about create a new function because it seems to have a lot of duplicates